crates/duty-tracker/src/contract_persister.rs

```diff
@@ -47,11 +47,11 @@ impl ContractPersister {
             // TODO(proofofkeags): make state not opaque at the DB level
             r#"
             CREATE TABLE IF NOT EXISTS contracts (
-                deposit_txid CHAR(64) PRIMARY KEY,
+                deposit_txid TEXT PRIMARY KEY,
```

Comment:
why?

Reply:
To make consistent with all other tables, but let's move discussion to #134